packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -14,8 +14,8 @@
 import * as vscode from "vscode";
 import * as fs from "fs";
 import * as globals from "../globals";
-import { Session, IProfile, IProfileLoaded, ProfileInfo, IConfigLayer } from "@zowe/imperative";
```

Comment:
Seems like `IConfigLayer` is still needed in this file 😋 
Otherwise, builds are failing 😅